src/main/java/mongodb/Main.java

```diff
@@ -0,0 +1,220 @@
+package mongodb;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.MongoException;
+import com.mongodb.client.*;
+import com.mongodb.client.model.*;
+import com.mongodb.client.result.DeleteResult;
+import com.mongodb.client.result.InsertManyResult;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.codecs.pojo.PojoCodecProvider;
+import org.bson.conversions.Bson;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.mongodb.client.model.Filters.*;
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+public class Main {
+  public static void main(String[] args) {
+    // TODO:
+    // Replace the placeholder connection string below with your
+    // Altas cluster specifics. Be sure it includes
+    // a valid username and password! Note that in a production environment,
+    // you do not want to store your password in plain-text here.
+    ConnectionString mongoUri = new ConnectionString("<Your Atlas Connection String>");
+
+    // Provide the name of the database and collection you want to use.
+    // If they don't already exist, the driver and Atlas will create them
+    // automatically when you first write data.
+    String dbName = "myDatabase";
+    String collectionName = "recipes";
+
+    // a CodecRegistry tells the Driver how to move data between Java POJOs (Plain Old Java Objects) and MongoDB documents
+    CodecRegistry pojoCodecRegistry = fromRegistries(MongoClientSettings.getDefaultCodecRegistry(),
+            fromProviders(PojoCodecProvider.builder().automatic(true).build()));
+
+    // The MongoClient defines the connection to our MongoDB datastore instance (Atlas) using MongoClientSettings
```

Comment:
You might want to mention that you're using a builder to instantiate the MongoClientSettings, and specify the codecRegistry and uri there.

Reply:
+1, adding this.